src/Adapter/Presenter/AbstractLazyArray.php

```diff
@@ -142,6 +142,28 @@ public function appendArray($array)
         }
     }
 
+    /**
+     * Set array key and callable from $array into the LazyArray.
+     * The callables are arrays where the elements are corresponding to the parameters of the call_user_func_array function
+     *
+     * @param array $array
+     */
+    public function appendLazyArray($array)
+    {
+        foreach ($array as $key => $value) {
+            // do not override any existing method
```

Comment:
I can't find where this method is used? :thinking:

Reply:
This method is not used in the PR but is an addition to allow module developper to add "Lazy" values to an lazy array, like I showed it here : https://github.com/PrestaShop/PrestaShop/pull/16093/commits/9c32593c871a6d7e41eb1e87195ef4650c191467